requirements.txt

```diff
@@ -23,7 +23,7 @@ pydantic==1.10.8
 #pymilvus==2.2.8
 pytest==7.2.2
 python_docx==0.8.11
-PyYAML==6.0
```

Comment:
can you keep the pinged version? add the version of "pip freeze |grep <pkg>" back into requirement.txt